assets/js/components/Dashboard/DatePicker.tsx

```diff
@@ -1,43 +1,37 @@
 import moment from "moment";
 import React, { useRef, useState } from "react";
-import { Form, Overlay, Popover } from "react-bootstrap";
+import { Button, Form, InputGroup, Overlay, Popover } from "react-bootstrap";
+import { CalendarCheck } from "react-bootstrap-icons";
 import Calendar from "react-calendar";
 
 interface DatePickerProps {
   selectedDate: string;
   onChange: (date: string) => void;
-  minDateString?: string;
-  maxDateString?: string;
   id?: string;
 }
 
-const DatePicker = ({
-  selectedDate,
-  onChange,
-  minDateString,
-  maxDateString,
-  id,
-}: DatePickerProps) => {
+const DatePicker = ({ selectedDate, onChange, id }: DatePickerProps) => {
   const [showOverlay, setShowOverlay] = useState(false);
   const ref = useRef(null);
-  const minDate = minDateString ? new Date(minDateString) : undefined;
-  const maxDate = maxDateString ? new Date(maxDateString) : undefined;
 
   return (
     <>
-      <Form.Control
-        id={id}
-        ref={ref}
-        className="date-picker"
-        name={`${id}-input`}
-        value={selectedDate}
-        onChange={(input) => onChange(input.target.value)}
-        onKeyUp={(e) => {
-          if (e.key === "Enter") {
-            setShowOverlay(true);
-          }
-        }}
-      />
+      <InputGroup className="date-picker" ref={ref}>
+        <Form.Control
+          id={id}
+          name={`${id}-input`}
+          value={selectedDate}
+          onChange={(input) => onChange(input.target.value)}
```

Comment:
`input` seems like an inaccurate name for this argument — usually it's called `event`, as it's the [event object](https://developer.mozilla.org/en-US/docs/Web/API/Event) for the change event. (The `event.target` is what would actually be the input element.)